cmd/snap/cmd_snap_op_test.go

```diff
@@ -444,7 +444,7 @@ func (s *SnapOpSuite) TestRevertRunthrough(c *check.C) {
 	c.Assert(rest, check.DeepEquals, []string{})
 	// tracking channel is "" in the test server
 	c.Check(s.Stdout(), check.Equals, `foo reverted to 1.0
-Snap foo is no longer tracking .
+ for foo is closed; temporarily forwarding to potato.
```

Comment:
Hmmm? This looks truncated

Reply:
as the comment above it says, it's because the test server replies with empty tracking channel (and it's not (yet) worth fixing)